Dockerfile

```diff
@@ -18,6 +18,10 @@ RUN apt-get --no-install-recommends -y install \
     libmozjs185-dev \
     python
 
+# Install Nodejs
+RUN curl -sL https://deb.nodesource.com/setup | bash -	
```

Comment:
@MCyprien you didn’t do anything wrong, no worries! :)

It’s just that we have to think about beyond just having someone interactively installing Node.js for CouchDB, hence our comments.

The best way forward would be to import the file as a dependency (needs an update in the NOTICE and LICENSE files) and then call it locally.